---
source: cli/tests/snapshot/main.rs
expression: out
---
testing foo/0...FAILED
testing foo/1...FAILED
testing foo/2...FAILED
testing foo/3...FAILED
test foo/0 succeeded (evaluated to 1), but it should have failed
test foo/1 failed, but the error didn't contain "wrong message". Actual error:
error: dynamic type error
  ┌─ [INPUTS_PATH]/doctest/fail_expected_error.ncl:1:7
  │
1 │ foo + "1"
  │       ^^^ this expression has type String, but Number was expected
  │
  = (+) expects its 2nd argument to be a Number


test foo/2 succeeded (evaluated to 1), but it should have failed
test foo/3 succeeded (evaluated to 1), but it should have failed
